spirit_simulator/a1_description/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+cmake_minimum_required(VERSION 2.8.3)
```

Comment:
General comment: perhaps spirit simulator would no longer make sense as the name of the folder. Maybe we can change it to just simulator? 
@jcnorby Not sure if we want to do it here or in a separate pr

Reply:
It isn't called spirit_simulator anymore, it should be quad_simulator. these files should be relocated accordingly.